### ISSUE TITLE
Implement 'extends' directive for installers

### ISSUE THREAD

**Issue by sixsixfive** [lutris/lutris#390]
so i reinstalled freespace 2 and had 4 entries for it i removed 3 of them but now i need to reinstall everything because fs2mp req: fs2open and fs2open req fs2gog which req fs2wine :/

**Comment by strycore** [comment]
I think it's something wrong with the installers themselves, hiding games won't change the fact that these installers make an incorrect use of "requires".
And it might be possible that writing the wanted installer is impossible until "extends" is added to Lutris scripts.

**Comment by strycore** [comment]
The Freespace 2 should look something like that: https://lutris.net/games/install/2863/view 
I can't test it since I don't have the game on GOG but it's basically all 4 of your installers merged into one.
Requires is meant to be used for mods or expansions that can actually be played (the Quake 1 and 2 mission packs are good examples of use cases for `requires`).

I'll try to ship the `extends` directive with 0.4.1 which will provide an add-on feature to scripts which will not create more game entries in the client.

**Comment by strycore** [comment]
Another directive we could provide is an `include` one, which would be the reverse of `extends`. This would allow running a separate script from within a script. If we also add a user choice command, this would allow to give more options during the install process. 

For example, the user could be prompted to choose between the GOG or the Steam version of a game, then when the selected game is installed, it would finish the script by setting up the game engine.

**Comment by sixsixfive** [comment]
>The Freespace 2 installer should look something like that: https://lutris.net/games/install/2863/view 

ugh, thats a terrible solution

at first it would break everything if a rew binary gets released and secondly remember this one: https://github.com/lutris/lutris/issues/351

also i would prefer to keep native and windows stuff seperated, some systems can also not use the mediavps due their hw limitations

**Comment by strycore** [comment]
I agree, it's not optimal but a different behavior won't be possible until `extends` is implemented.
As for the Windows version, it shouldn't be in the client if it's not meant to be played. (With all the modifications in the installer, I'm not even sure the GOG version works).  It seems the only purpose of this installer is to provide files for the fs2open installer. In that case, both fs2open and fs2gog should be merged until `include` is implemented and there is a way of flagging installers as data sources only.

**Comment by strycore** [comment]
extends is now available
